web/app/loading.tsx

```diff
@@ -0,0 +1,13 @@
+import './styles.css'
+
+function LoadingDots() {
```

Comment:
Where is this function used?

Reply:
In the past when I have used instant loading states (https://nextjs.org/docs/app/building-your-application/routing/loading-ui-and-streaming#instant-loading-states) I have implemented them just like this. Reading the link though, I see that best practices is to wrap each page in a <Suspense> tag and pass the loading function as the fallback. 

I have been dealing with some issues with my mac today which has prevented me from being able to build the server. Will deal with this tomorrow and push the changes.